packages/braid-design-system/src/lib/components/ButtonIcon/ButtonIcon.test.tsx

```diff
@@ -17,8 +17,8 @@ describe('ButtonIcon', () => {
         <BraidTestProvider>
           {/*
             Rendering `PrivateButtonIcon` so no tooltip is added.
-            Popper tooltip library does not like being rendered
-            to static markup, but not required for this test.
+            Tooltip does not like being rendered to static markup,
+            but not required for this test.
```

Comment:
Given the new tests below are using `ButtonIcon`, lets try updating this as well. If it still doesnt work itd be worth updating the comment to reflect the constraint of using `html-validate/jest` for this test.